src/commands/dev/mod.rs

```diff
@@ -42,7 +43,11 @@ pub fn dev(
     ip: Option<&str>,
     verbose: bool,
 ) -> Result<(), failure::Error> {
-    message_box::dev_alpha_warning();
+    let wrangler_dev_msg = style("`wrangler dev`").yellow().bold();
+    let feedback_url = style("https://github.com/cloudflare/wrangler/issues/1047")
+        .blue()
+        .bold();
+    message::boxx(&format!("{0} is currently unstable and there are likely to be breaking changes!\nFor this reason, we cannot yet recommend using {0} for integration testing.\n\nPlease submit any feedback here: {1}", wrangler_dev_msg, feedback_url));
```

Comment:
you had talked about naming this "billboard"?

Reply:
yah i'll rename it client side for now and then i'm going to rename the actual crate sometime later and then we can update that in a subsequent pr